lib/menu.rb

```diff
@@ -0,0 +1,48 @@
+require 'twilio-ruby'
+
+attr_reader :menu, :order, :order_total
+account_sid = "ACef87d7dfc16f3cebd55fc9ec9969335a"
+
+class Menu
+
+  def initialize
+    @menu = { "hamburger" => 2, "pizza" => 5 }
+    @order = []
+    @order_total = []
+  end
+
+  def check_menu
+    menu
+  end
+
+  def select_dishes(*selection, total)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for select_dishes is too high. [21.1/15] (http://c2.com/cgi/wiki?AbcMetric)